File: app/src/main/java/ru/androidacademy/ateam/model/AppDataBase.java
package ru.androidacademy.ateam.model;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;
import ru.androidacademy.ateam.model.dao.DeckDao;
import ru.androidacademy.ateam.model.dao.WordDao;
import ru.androidacademy.ateam.model.tables.Deck;
import ru.androidacademy.ateam.model.tables.Word;

@Database(entities = {Word.class, Deck.class}, version = 1)
public abstract class AppDataBase extends RoomDatabase {
    public abstract WordDao wordDao();
    public abstract DeckDao deckDao();
}

File: app/src/main/java/ru/androidacademy/ateam/db/AppDatabase.kt
package ru.androidacademy.ateam.db

import androidx.room.Database
import androidx.room.RoomDatabase
import ru.androidacademy.ateam.db.dao.DeckDao
import ru.androidacademy.ateam.db.dao.WordDao
import ru.androidacademy.ateam.model.Deck
import ru.androidacademy.ateam.model.Word


@Database(entities = [Word::class, Deck::class], version = 1, exportSchema = false)
abstract class AppDatabase : RoomDatabase() {

    abstract fun wordDao(): WordDao
    abstract fun deckDao(): DeckDao

}